common/src/boot_status.rs

```diff
@@ -62,8 +62,9 @@ pub enum RomBootStatus {
     FmcAliasDerivationComplete = FMCALIAS_BOOT_STATUS_BASE + 5,
 
     // Cold Reset Statuses
-    ColdResetStarted = COLD_RESET_BOOT_STATUS_BASE,
-    ColdResetComplete = COLD_RESET_BOOT_STATUS_BASE + 1,
+    KatStarted = COLD_RESET_BOOT_STATUS_BASE,
```

Comment:
KATs are not specific to Cold Boot. Can you add the following global:
const ROM_GLOBAL_BOOT_STATUS_BASE: u32 = 385;

And in RomBootStatus num:
    // Rom Global Statuses
    KatStarted = ROM_GLOBAL_BOOT_STATUS_BASE,
    KatComplete = ROM_GLOBAL_BOOT_STATUS_BASE + 1,

Reply:
The enumeration is using a value greater than the ColdResetStarted.  As per current implementation, we are using monotonically increasing boot status code to indicate order of events.

Reply:
Done